src/PageContent/AssignVolunteers/Edit.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import moment from 'moment';
+import { StringFormat } from '../../Enums';
+
+class Edit extends Component {
+
+    constructor(props) {
+        super(props);
+        let date = new Date(this.props.delivery.date);
+        this.state = {
+            day: moment(date).format('dddd'),
+            date: moment(date).format('l'),
+            from: this.props.delivery.startTime,
+            to: this.props.delivery.endTime,
+            donatingAgency: this.props.delivery.donatingAgency.agency,
+            receivingAgency: this.props.delivery.receivingAgency.agency,
+            deliverer1: {
+                name: '',
+                phone: '',
+                email: ''
+            },
+            deliverer2: {
+                name: '',
+                phone: '',
+                email: ''
+            }
+        };
+    }
+
+    render() {
+        return (
+            <div>
+                <div className="date-container">
+                    {this.state.day} {this.state.date} Pick-up between {this.state.from} - {this.state.to}
+                </div>
+                <div className="content-container">
+                    <h5 className="location" id="top"><i className="fas fa-circle circle-marker" /><span className="spacing" />{this.state.donatingAgency}</h5>
+                    <i className="fas fa-circle ellipses" />
+                    <i className="fas fa-circle ellipses" />
+                    <i className="fas fa-circle ellipses" />
+                    <h5 className="location" id="bottom"><i className="fas fa-map-marker-alt map-marker" /><span className="spacing" />{this.state.receivingAgency}</h5>
+                </div>
+                <div className="form-container">
+                    <form onSubmit={this.handleConfirmClick.bind(this)}>
+
+                        <label className="label-component">Student Deliverers</label>
+
+                        <div className="form-parent">
+
+                            <div className="form-child">
+                                <label className="label-component details">Student 1</label><br />
+                                <input name="name1" type="text" className="form-input" defaultValue={this.props.delivery.delivererGroup.deliverers[0] ? this.props.delivery.delivererGroup.deliverers[0].name : ''} required/><br />
```

Comment:
for readability, you can do what you're doing with `donatingAgency`, you could add
```
deliverers: this.props.delivery.delivererGroup.deliverers
```
to `state` in constructor
and then use `this.state.deliverers[0] ? ...`

Reply:
Fixed.